src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests/Handlers/Slider/SliderHandlerTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Threading.Tasks;
+using Xamarin.Platform.Handlers.DeviceTests.Stubs;
+using Xunit;
+
+namespace Xamarin.Platform.Handlers.DeviceTests
+{
+	public partial class SliderHandlerTests : HandlerTestBase<SliderHandler>
+	{
+#if __ANDROID__
+		[Fact(Skip = "Currently Fails on Android")]
```

Comment:
I used the issue here as an initial use case for writing a test

https://github.com/dotnet/maui/issues/260

Once we merge this code the current PR to fix the above issue can rebase and then uncomment this test